packages/communicator-bcrypt/tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "target": "esnext",                                  /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
     "experimentalDecorators": true,                      /* Enable experimental support for legacy experimental decorators. */
     "emitDecoratorMetadata": true,                       /* Emit design-type metadata for decorated declarations in source files. */
-    "module": "commonjs",                                /* Specify what module code is generated. */
+    "module": "Node16",                                /* Specify what module code is generated. */
```

Comment:
We should use tsconfig's [`extend`](https://www.typescriptlang.org/tsconfig#extends) option to centralize our config settings rather than duplicating the settings in each tsconfig.json file

Reply:
Follow-on?

Reply:
tsconfig supports extension/hierarchy. You can define a base tsconfig file in the repo root and then reference it in all the other tsconfig files via `extend` property. That way we can centralize all the settings that are the same but still customize individual packages as needed

Reply:
That's a good idea. Probably a separate PR though.